utils/prepare_puppeteer_core.js

```diff
@@ -22,4 +22,5 @@ const packagePath = path.join(__dirname, '..', 'package.json');
 const json = require(packagePath);
 
 json.name = 'puppeteer-core';
+delete json.scripts['install'];
```

Comment:
```suggestion
delete json.scripts.install;
```

Reply:
I applied this commit but it looks like twp CI checks fail now.

Reply:
It works as expected when testing locally. Can it be something intermittent?

Reply:
These are flaky tests, unrelated to your changes. Don't worry about it!